src/Todo/Bundle/UserBundle/Resources/config/services.yml

```diff
@@ -0,0 +1,4 @@
+services:
```

Comment:
The `services.yml` is not used as you see. So delete it and remove its import line from `config.yml` file